packages/js/src/utils/errors.ts

```diff
@@ -18,6 +18,14 @@ const logtoErrorCodes = Object.freeze({
   },
 });
 
+const callbackUriErrorCodes = Object.freeze({
+  redirectUriMismatched: 'Redirect uri mismatched',
```

Comment:
Shouldn't we capitalize the uri? `uri` => `URI`

Reply:
URI is better. Fixed, PTAL.